pro/src/components/IndividualOfferForm/constants.ts

```diff
@@ -3,9 +3,21 @@ import { CATEGORIES_DEFAULT_VALUES } from './Categories/constants'
 import { EXTERNAL_LINK_DEFAULT_VALUES } from './ExternalLink/constants'
 import { INFORMATIONS_DEFAULT_VALUES } from './Informations/constants'
 import { NOTIFICATIONS_DEFAULT_VALUES } from './Notifications/constants'
+import { OFFER_LOCATION_DEFAULT_VALUES } from './OfferLocation/constants'
 import { USEFUL_INFORMATIONS_DEFAULT_VALUES } from './UsefulInformations/constants'
 
 export const FORM_DEFAULT_VALUES = {
+  ...INFORMATIONS_DEFAULT_VALUES,
+  ...CATEGORIES_DEFAULT_VALUES,
+  ...USEFUL_INFORMATIONS_DEFAULT_VALUES,
+  ...ACCESSIBILITY_DEFAULT_VALUES,
+  ...NOTIFICATIONS_DEFAULT_VALUES,
+  ...EXTERNAL_LINK_DEFAULT_VALUES,
+  ...OFFER_LOCATION_DEFAULT_VALUES,
+  isDuo: false,
+}
```

Comment:
Pour être sûr que ça ne casse pas
```suggestion
export const FORM_DEFAULT_VALUES = {
  ...FORM_DEFAULT_VALUES_NO_OFFER_LOCATION
  ...OFFER_LOCATION_DEFAULT_VALUES,
  isDuo: false,
}
```